docs/index.md

```diff
@@ -0,0 +1,34 @@
+(index-label)=
+
+# Plone Distribution Support
```

Comment:
We use sentence casing for headings, with title casing for proper nouns and uppercasing for acronyms.

```suggestion
# Plone distribution support
```